threedi_modelchecker/config.py

```diff
@@ -1176,221 +1142,215 @@ def is_none_or_empty(col):
         ),
-        message="v2_global_settings.use_2d_flow may not be TRUE if no dem file is provided",
+        message="model_settings.use_2d_flow may not be TRUE if no dem file is provided",
     ),
     QueryCheck(
         error_code=320,
-        column=models.GlobalSetting.use_2d_flow,
-        invalid=Query(models.GlobalSetting).filter(
-            first_setting_filter,
-            models.GlobalSetting.use_1d_flow == False,
-            models.GlobalSetting.use_2d_flow == False,
+        column=models.ModelSettings.use_2d_flow,
+        invalid=Query(models.ModelSettings).filter(
+            models.ModelSettings.use_1d_flow == False,
+            models.ModelSettings.use_2d_flow == False,
         ),
-        message="v2_global_settings.use_1d_flow and v2_global_settings.use_2d_flow cannot both be FALSE",
+        message="model_settings.use_1d_flow and model_settings.use_2d_flow cannot both be FALSE",
     ),
     QueryCheck(
         level=CheckLevel.WARNING,
         error_code=321,
-        column=models.GlobalSetting.manhole_storage_area,
-        invalid=Query(models.GlobalSetting).filter(
-            first_setting_filter,
-            models.GlobalSetting.manhole_storage_area > 0,
+        column=models.ModelSettings.manhole_aboveground_storage_area,
+        invalid=Query(models.ModelSettings).filter(
+            models.ModelSettings.manhole_aboveground_storage_area > 0,
             (
-                (models.GlobalSetting.use_2d_flow == True)
-                | (~is_none_or_empty(models.GlobalSetting.dem_file))
+                (models.ModelSettings.use_2d_flow == True)
+                | (~is_none_or_empty(models.ModelSettings.dem_file))
             ),
         ),
-        message="sub-basins (v2_global_settings.manhole_storage_area > 0) should only be used when there is no DEM supplied and there is no 2D flow",
+        message="sub-basins (model_settings.manhole_aboveground_storage_area > 0) should only be used when there is no DEM supplied and there is no 2D flow",
     ),
     QueryCheck(
         error_code=322,
-        column=models.GlobalSetting.water_level_ini_type,
-        invalid=Query(models.GlobalSetting).filter(
-            first_setting_filter,
-            ~is_none_or_empty(models.GlobalSetting.initial_waterlevel_file),
-            models.GlobalSetting.water_level_ini_type == None,
+        column=models.InitialConditions.initial_water_level_aggregation,
+        invalid=Query(models.InitialConditions).filter(
+            ~is_none_or_empty(models.InitialConditions.initial_water_level_file),
+            models.InitialConditions.initial_water_level_aggregation == None,
         ),
-        message="an initial waterlevel type (v2_global_settings.water_level_ini_type) should be defined when using an initial waterlevel file.",
+        message="an initial waterlevel type (initial_settings.initial_water_level_aggregation) should be defined when using an initial waterlevel file.",
```

Comment:
Please go through the rest with a find-replace

```suggestion
        message="an initial waterlevel type (initial_conditions.initial_water_level_aggregation) should be defined when using an initial waterlevel file.",
```

Reply:
Thanks for picking this up. Did a sed on all py files and all should be replaced; see 4a674d3207f04e0364d97aa81950f1b65c17c1d6